openlibrary/core/vendors.py

```diff
@@ -409,7 +414,7 @@ def clean_amazon_metadata_for_load(metadata: dict) -> dict:
 
 
 def create_edition_from_amazon_metadata(
-    id_: str, id_type: Literal['asin', 'isbn'] = 'isbn'
+    id_: str, id_type: Literal['asin', 'isbn'] = 'isbn', retries: int = 0
```

Comment:
I think that `retries` needs to be passed to the `get_amazon_metadata` call on line 428, below.

Reply:
Thanks,  Done.